.gitlab-ci.yml

```diff
@@ -46,6 +46,13 @@
   script:
     - bundle exec rake spec
 
+# To avoid running a prohibitive number of tests every commit,
+# don't set this env var in your gitlab instance
+.only_with_SIMP_FULL_MATRIX: &only_with_SIMP_FULL_MATRIX
```

Comment:
Please make sure at least one FIPS test and at least one OEL test is included in only_with_SIMP_FULL_MATRIX